library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -512,14 +533,34 @@ void vtkF3DRenderer::SetHDRIFile(const std::string& hdriFile)
     this->TextActorsConfigured = false;
     this->RenderPassesConfigured = false;
 
+    this->HasValidHDRIHash = false;
+    this->HasValidHDRITexture = false;
+    this->HasValidHDRISH = false;
+    this->HasValidHDRISpec = false;
+
     this->HDRITextureConfigured = false;
-    this->HDRILUTConfigured = false;
     this->HDRISphericalHarmonicsConfigured = false;
     this->HDRISpecularConfigured = false;
     this->HDRISkyboxConfigured = false;
   }
 }
 
+//----------------------------------------------------------------------------
+void vtkF3DRenderer::SetUseImageBasedLighting(bool use)
+{
+  if (use != this->GetUseImageBasedLighting())
+  {
+    this->Superclass::SetUseImageBasedLighting(use);
```

Comment:
I could define my own variable instead of relying on UseImageBasedLighting, but that doesnt seems necessary